actions/retest/main.go

```diff
@@ -142,7 +142,16 @@ func main() {
 					continue
 				}
 
+				// get latest version of each status.
```

Comment:
please put this in in a helper function describing why it is needed (similar to the commit message)